src/rt/sections_android.d

```diff
@@ -63,11 +63,11 @@ void initSections() nothrow @nogc
 {
     pthread_key_create(&_tlsKey, null);
 
-    auto mbeg = cast(immutable ModuleInfo**)&__start_minfo;
-    auto mend = cast(immutable ModuleInfo**)&__stop_minfo;
+    auto mbeg = cast(immutable ModuleInfo**)&__start___minfo;
+    auto mend = cast(immutable ModuleInfo**)&__stop___minfo;
```

Comment:
This may help: https://www.airs.com/blog/archives/56
We set the section name to `"__minfo"`, so that's where the extra underscores come from: https://github.com/ldc-developers/ldc/blob/0b8d09d517ce2155496bf9e8f39ae57eff0b6ad0/gen/modules.cpp#L375